source/frontend/src/components/maps/leaflet/Layers/PointClusterer.tsx

```diff
@@ -224,31 +183,29 @@ export const PointClusterer: React.FC<React.PropsWithChildren<PointClustererProp
 
       if (groupBounds.isValid()) {
```

Comment:
does this if block have a purpose if fitbounds is removed?

Reply:
It also updates the filterContext. I dont think is doing much right now. But that would be another task to remove the filtercontext imo.